src/helpers/Utils/Utils.js

```diff
@@ -187,14 +187,24 @@ export const normalizeLanguageName = (language) => {
 };
 
 /**
- * Given array of objects with `id` property, map to object w/ properties by `id`
+ * Given an array of objects where each object has a property whose key
```

Comment:
I think the first line should be a standalone sentence, much like Python docstrings, git commit messages, etc..  But I don't care more about that than I care about finishing this discussion.